lib/appium.js

```diff
@@ -28,7 +28,11 @@ class AppiumDriver extends BaseDriver {
     // the main Appium Driver has no new command timeout
     this.newCommandTimeoutMs = 0;
 
-    this.args = args;
+    this.args = Object.assign({}, args);
+    if (this.args.relaxedSecurityEnabled) {
+      this.relaxedSecurityEnabled = true;
+      delete this.args.relaxedSecurityEnabled;
```

Comment:
Why this extra step?

Reply:
because this option should not be in capabilities. One anyway cannot change it from there

Reply:
But only the default capabilities from the command line arguments make it into the caps sent to the driver (https://github.com/appium/appium/blob/master/lib/appium.js#L176-L177)

Reply:
ok, makes sense